oneanddone/tests/functional/pages/home.py

```diff
@@ -51,3 +63,27 @@ def click_pick_a_task_button(self):
     def click_available_tasks(self):
         self.find_element(self._available_tasks_locator).click()
         return AvailableTasksPage(self.selenium, self.base_url).wait_for_page_to_load()
+
+    def click_create_task(self, task_name):
```

Comment:
This is doing more than just clicking create task. It's not a problem to have a function that combines several interactions, but we should make sure the name is clear. In this case I think we're doing too much in the page object, and if we break these interactions down we could call them directly from the test.